apps/export/tests/test_apis.py

```diff
@@ -56,3 +56,17 @@ def test_trigger_api_with_export_permission(self):
         export = Export.objects.get(id=response.data['export_triggered'])
         self.assertTrue(export.pending)
         self.assertEqual(export.exported_by, self.user)
+
+    def test_delete_export(self):
+        export = self.create(Export, exported_by=self.user)
+        url = '/api/v1/exports/{}/'.format(export.id)
+
+        self.authenticate()
+        response = self.client.delete(url)
+        self.assert_204(response)  # delete from api
+
+        # check for database
+        assert Export.objects.count() == 1  # should not delete from database
```

Comment:
Instead of `1` calculate the count before DELETE request and then make sure they are the same.